activation/handler.go

```diff
@@ -548,18 +534,40 @@ func (h *Handler) HandleGossipAtx(ctx context.Context, peer p2p.Peer, msg []byte
 	if errors.Is(err, errKnownAtx) && peer == h.local {
 		return nil
 	}
+
+	// broadcast malfeasance proof last as the verification of the proof will take place
+	// in the same goroutine
+	if proof != nil {
+		gossip := types.MalfeasanceGossip{
+			MalfeasanceProof: *proof,
+		}
+		encodedProof, err := codec.Encode(&gossip)
+		if err != nil {
+			h.log.With().Fatal("failed to encode malfeasance gossip", log.Err(err))
```

Comment:
maybe MustEncode, since the code was changed

Reply:
Fixed